examples/run/alpaka/throughput_st.cpp

```diff
@@ -16,9 +16,8 @@ int main(int argc, char* argv[]) {
 
     // Execute the throughput test.
     static const bool use_host_caching = true;
-    return traccc::throughput_st<
-        traccc::alpaka::full_chain_algorithm,
-        traccc::alpaka::vecmem_resources::host_memory_resource>(
+    return traccc::throughput_st<traccc::alpaka::full_chain_algorithm,
+                                 vecmem::host_memory_resource>(
```

Comment:
The only caveat of the Pimpl implementation is that this is now using the default `vecmem::host_memory_resource` rather than in the older case it was using the most appropriate vecmem resource (i.e. `cuda|hip|sycl`).

Am I missing something to enable this?